### DISCUSSION TITLE
lang.from_bytes does not recreate the pipeline properly

### DISCUSSION THREAD

**Discussion by oliviercwa** [explosion/spaCy#4964] [Help: Coding & Implementations] [answered]
I need to load and save pipeline as bytes. When using the following example, the pipeline that is being loaded does not have its pipeline components set.

What is the right way of saving a custom pipeline to a stream and loading it back?

Thank you,
Olivier

## How to reproduce the behaviour
```python
nlp = spacy.load('en_core_web_md')
with open('c\\test', 'wb') as outfile:
  outfile.write(nlp.to_bytes())

# This one will detect entities properly
test = nlp('I am going to Germany')

nlp = spacy.blank('en')
with open('c:\\test', 'rb') as outfile:
  nlp.from_bytes(outfile.read())

# NER has not been loaded.
test = nlp('I am going to Germany')
```

## Your Environment
* Operating System: Windows 10
* Python Version Used: 3.7.4
* spaCy Version Used: 2.1.9
* Environment Information:

**Comment by adrianeboyd** [comment, marked as answer]
See: https://spacy.io/usage/saving-loading#pipeline
